tests/support/pytest/helpers.py

```diff
@@ -187,6 +187,48 @@ def test_blah():
     )
 
 
+@pytest.helpers.register
+def temp_state_directory(name, saltenv="base"):
+    """
```

Comment:
As I mentioned above, I'd for us not to use `RUNTIME` under pytest, we now have the power of fixtures.

We could create an improved fixture regarding the state tree so we don't have to pass around `base_env_state_tree_root_dir` all the time... I'll think about that for a bit....

Reply:
Remove the helper?

Reply:
Or are you using it in `unittest` test?